cypress/integration/step_definitions/Ui/single_vanities.cy.js

```diff
@@ -31,3 +31,125 @@ When('User clicks to Single Vanities', () => {
 
 
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+//Scenario: Add to Cart Button & Your Cart page functionality test
+
+let singleVanitiesPage=null
+before(()=>{
+    singleVanitiesPage =new SingleVanitiesPage();
+})
+
+And ('User clicks to random add-to-cart button' , () =>{
+    singleVanitiesPage.addToCartRandomProduct() 
+})
+
+And('it should direct to Your Cart page', () => {
+
+    const expectedTitle = 'Your Shopping Cart'; 
+    cy.title().should('contain', expectedTitle);
+  }); 
+
+And ('the added product should appear in Your Cart page' , () => {  
+   
+    const addedProduct = '.cart-item'; 
+    cy.get(addedProduct).should('exist');
+});
+
+And ('user clicks to Check-Out Button' , () => {
+    singleVanitiesPage.clickCheckOutButton()
+})
+
+Then ('it should direct to the Payment' , () => {
+    const expectedTitle = 'Paiement'; 
```

Comment:
What about using this common function here: [https://github.com/abeyis/Avaluxury/blob/38c2d844ab8a1c86230f3ff74aafe2c12996e605/cypress/integration/step_definitions/Ui/commonSteps.js#L9](url)

Reply:
Done. @KubraSA